core-dev/packages/coq.dev/opam

```diff
@@ -1,23 +1,55 @@
-opam-version: "1.1"
-maintainer: "dev@clarus.me"
-homepage: "http://coq.inria.fr/"
-bug-reports: "https://coq.inria.fr/bugs/"
-license: "LGPL 2"
+opam-version: "1.2"
+maintainer: "coqdev@inria.fr"
+authors: "The Coq development team, INRIA, CNRS, University Paris Sud, University Paris 7, Ecole Polytechnique."
+homepage: "https://coq.inria.fr/"
+bug-reports: "https://github.com/coq/coq/issues"
+dev-repo: "https://github.com/coq/coq.git"
+license: "LGPL-2.1"
 build: [
-  ["./configure"
+  [
+    "./configure"
     "-configdir" "%{lib}%/coq/config"
-    "-mandir" man
-    "-with-doc" "no"
     "-prefix" prefix
+    "-mandir" man
+    "-docdir" doc
     "-libdir" "%{lib}%/coq"
     "-datadir" "%{share}%/coq"
     "-camlp5dir" "%{lib}%/camlp5"
     "-coqide" "no"
   ]
   [make "-j%{jobs}%"]
+  [make "-j%{jobs}%" "byte"]
+]
+install: [
   [make "install"]
+  [make "install-byte"]
+]
+remove: [
+  ["rm" "-R" "%{lib}%/coq" "%{share}%/coq"]
+  ["rm"
+  "%{man}%/man1/coqc.1"
+  "%{man}%/man1/coqchk.1"
+  "%{man}%/man1/coqdep.1"
+  "%{man}%/man1/coqdoc.1"
+  "%{man}%/man1/coqide.1"
```

Comment:
If there is -coqide no, why to remove `coqide.1`? Isn't there a "bug" in the install target?

Reply:
Hm, good question. I think that indeed the `coq` opam package does install this manpage, let me check again.

Reply:
Indeed, this is the case. So for now, the opam package is correct -- the `uninstall` should un-do the `install`, after all.

I will open an issue about the `coqide` man page.